samples/benchmarks/template_engine.py

```diff
@@ -26,12 +26,13 @@ def gen():
 async def render():
     template = engine.loaded_templates['test']
     c = engine.prepared_calls[template]
-    t1 = time.time()
+    t2 = time.time()
```

Comment:
What exactly is the rationale behind this change (and all the similar ones that follow)?

Reply:
@rubik: That's part of https://github.com/vibora-io/vibora/pull/71/commits/a48ca8de9f1887a39eef165ad91a5a45d7df90d7; just, as the title says, avoiding naming a variable the same thing in two different and adjacent scopes, which can lead to confusion.